### PR TITLE
Created 2 files and putted tag <Right/> in Main.js

### PR DESCRIPTION
**Pull request by hideki0324**

### CODE REVIEW

**Comment by hideki0324** [comment]
yeah, I checked my file. you're right..... It's a mistyped. Thanks.

File: comps/Right.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import { removeProperties, removeTypeDuplicates } from '@babel/types';
+
+
+function Right(){
+    remote(
```
Comment (AndriusCvetkovas) [inline comps/Right.js]: supposed to be return

File: __tests__/Right-test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import 'react-native';
+import React from 'react';
+import Right from '../comps/Right';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
```
Comment (AndriusCvetkovas) [inline __tests__/Right-test.js]: it or if renders correctly?

Comment (hideki0324) [reply __tests__/Right-test.js]: I just copied them from App-test.js. So i think 'it' is correct.

File: comps/Right.js (hunk 1 above)
Comment (hideki0324) [reply comps/Right.js]: yeah, thanks

File: __tests__/Right-test.js (hunk 2 above)
Comment (AndriusCvetkovas) [reply __tests__/Right-test.js]: oh sorry my mistake

**Pull request closed without merging**